src/Informasjonsside/Informasjonsside.tsx

```diff
@@ -100,7 +106,7 @@ const Informasjonsside: FunctionComponent<RouteComponentProps> = props => {
                             </span>
                             <span className={cls.element('lenke')}>
                                 <EksternLenke href="https://www.altinn.no/hjelp/profil/roller-og-rettigheter/">
-                                    Les mer om roller og rettigheter på Altinn.no
+                                    Finn mer informasjon om roller og rettigheter på Altinn.no
                                 </EksternLenke>
                             </span>
                             <AltinnVideoModal isOpen={isVideoModalOpen} lukkModal={() => setVideoModalOpen(false)} />
```

Comment:
Nå har vi jo fjernet "Helse og sosial" fra teksten her og bare lister opp de tre enkeltrettighetene til arbeidstrening, mildertidig og varig lønnstilskudd, og det er fint det, men i videoen her viser jeg jo hvordan man både tildeler enkeltrettighet og rollen `Helse, sosial og velferdstjenester`. Er det dumt? 🤷‍♂️

Reply:
Jeg synes det er greit for nå. Vi kan endre videoen ved en senere anledning.